README.md

```diff
@@ -94,14 +93,40 @@ initializes folder structure
 ```
 <!-- END_HELP_COMMAND_INIT -->
 
-### Generate RBI files for gems
+## Usage
+
+### Generating RBI files for gems
 
-Command: `tapioca gem [gems...]`
+Sorbet does not read the code in your gem dependencies, so it does not know the constants and methods declared inside gems. Tapioca is able to load your gem dependencies from you application `Gemfile` and compile RBI files to represent their content.
 
-This will generate RBIs for the specified gems and place them in the RBI directory.
+To generate the RBI files for the gems used in your application run the following command:
+
+```sh
+$ bin/tapioca gems [gems...]
+
+Removing RBI files of gems that have been removed:
+
+  Nothing to do.
+
+Generating RBI files of gems that are added or updated:
+
+  Requiring all gems to prepare for compiling...    Done
+
+  Compiled ansi
+      create  sorbet/rbi/gems/ansi@1.5.0.rbi
+
+  ...
+
+All operations performed in working directory.
+Please review changes and commit them.
+```
+
+This will load your application, find all the gems required by it and generate an RBI file for each gem under the sorbet/rbi/gems directory. This will also import the signatures that can be found inside each gem source code as well as the YARD documentation of the gem.
```

Comment:
```suggestion
This will load your application, find all the gems required by it and generate an RBI file under the `sorbet/rbi/gems` directory for each of those gems. This process will also import signatures found inside gem sources, and, optionally, any YARD documentation inside the gem.
```